README.md

```diff
@@ -7,7 +7,7 @@ Luppp - OpenAV Productions
 
 This is the repository of Luppp, the live looping tool.
 
-![screenshot](https://raw.github.com/harryhaaren/openAV-Luppp/master/resources/screenshots/1.0.png "Luppp 1.0 Screenshot")
+![screenshot](resources/screenshots/1.1.png "Luppp 1.0 Screenshot")
```

Comment:
When I push the changes, I don't think this URL will resolve correctly - have you done this in previous repos that it worked? From previous experience, we need a full HTTP URL